packages/nextjs/services/web3/wagmiConnectors.tsx

```diff
@@ -70,7 +57,7 @@ export const wagmiConnectors = connectorsForWallets([
       braveWallet({ chains: appChains.chains }),
       coinbaseWallet({ appName: "scaffold-eth", chains: appChains.chains }),
       rainbowWallet({ chains: appChains.chains }),
-      burnerWalletConfig({ chains: burnerChains.chains }),
+      burnerWalletConfig({ chains: [appChains.chains[0]] }),
```

Comment:
Removed `burnerChains` and inlined it with `appChains.chains[0]`  because `configuredNetwork` will be 0th network and also other stuff from `burnerChains` are not used like `burnerChains.providers` .... etc